### PR TITLE
Edit metrics

### PR DESCRIPTION
**Pull request by sascala**
Thanks for your comments, @aquamatthias. I have one final question... and otherwise I'm done.

### CODE REVIEW

File: docs/docs/metrics.md [hunk 1]
```diff
@@ -117,37 +94,31 @@ Please note that prior to <span class="label label-default">v0.15</span> these w
...
 
-## Potential issues
+## Potential pitfalls
 
 ### Derived metrics (mean, p99, ...)
 
-Our metrics library calculates derived metrics like "mean" and "p99". Unfortunately, if reported to statsd, they
-do not only relate to the reporting interval but the whole live time of the app with some exponential weighting
-algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
+Our metrics library calculates derived metrics like "mean" and "p99." However, Marathon provides these metrics for the entire live of of the app and applies an exponential weighting algorithm as a heuristic. For more precise metrics, build your dashboard around "counts" rather than "rates" where possible.
```
Comment (sascala) [inline docs/docs/metrics.md]: I was guessing a little bit here.

File: docs/docs/metrics.md [hunk 2]
```diff
@@ -117,37 +94,31 @@ Please note that prior to <span class="label label-default">v0.15</span> these w
...
 
-### Statsd, derived statistics and metric names
+### Statsd, derived statistics, and metric names
 
-Statsd typically creates derived statistics (mean, P99) from what is reported. This might interact in a weird
-fashion with the derived statistics that our codahale metrics package reports.
+Statsd typically creates derived statistics (mean, p99) from mean values Marathon reports. Our codahale metrics package also reports derived statistics, which may differ from those created by statsd. Be sure you know where you are reporting and computing mean values.
```
Comment (sascala) [inline docs/docs/metrics.md]: What role to metric names play here? Is the trouble that there may be different statistics with the same metrics name?

Comment (aquamatthias) [inline docs/docs/metrics.md]: No the problem is: aggregation should happen in one place. The metrics library we use, already has this ability. Aggregation of aggregated values could lead to wrong metrics. So your description is correct.

**Comment by aquamatthias** [comment]
@sascala Thanks! LGTM. please squash and rebase.

**Comment by aquamatthias** [comment]
And also please squash your commits.

**Pull request merged**